Month1/Week2/Section1/Exercise1/main1.cpp

```diff
@@ -0,0 +1,42 @@
+/*Exercise 1 - Understanding classes
+Create a class called FirstClass with a public constructor and destructor. Have the constructor output �I�m in the constructor!� and have the destructor output �I�m in the destructor!�. Then, instantiate 5 classes and run your program, are the constructor and destructor calls where you expected them to be?
+
+Now create a loop of at least five iterations. For each iteration, instantiate your class once. Run your program, are the constructor and destructor calls where you expected them to be?
+
+*/
+#include <iostream>
+
+#include<string>
+
+class FirstClass
```

Comment:
From now, we're following the standards. Your classes should use the `F` prefix. I know the instructions don't reflect this, but as stated on the standard page in Tettra, you still need to adjust it to follow the standard. 

The exercises will be adjusted to use the standard over time.

**Please fix this for all the exercises**

Reply:
Done